lib/Dialect/TritonGPU/Transforms/RemoveLayoutConversions.cpp

```diff
@@ -171,47 +171,21 @@ static bool hasConvertToMMATransisitiveUse(Operation *op, Attribute encoding) {
                 dstEncoding.dyn_cast<triton::gpu::MmaEncodingAttr>())
           return (mmaLayout.getVersionMajor() > 1) ? true
                                                    : mmaLayout == encoding;
-        if (dstEncoding.isa<triton::gpu::DotOperandEncodingAttr>())
-          return encoding.cast<triton::gpu::MmaEncodingAttr>()
-                     .getVersionMajor() > 1;
-      }
-      auto yield = dyn_cast<scf::YieldOp>(op);
-      if (!yield)
-        continue;
-      auto forOp = dyn_cast<scf::ForOp>(yield.getOperation()->getParentOp());
-      if (!forOp)
-        continue;
-      for (OpOperand &operand : yield->getOpOperands()) {
-        Operation *def = operand.get().getDefiningOp();
-        if (def && forwardSlice.count(def) &&
-            (seen.insert(operand.get()).second == true))
-          queue.push_back(forOp.getRegionIterArg(operand.getOperandNumber()));
+        if (auto mfmaLayout =
```

Comment:
Can we rename the function name `hasConvertToMMATransisitiveUse` to be general?

Reply:
Both `MfmaEncodingAttr` and `NvidiaMmaEncodingAttr` have `MmaEncodingTrait`, so `MMA` here sounds genernal?